crates/common/consensus/src/deneb/beacon_state.rs

```diff
@@ -797,6 +797,15 @@ impl BeaconState {
         self.balances
             .push(amount)
             .map_err(|err| anyhow!("Couldn't push to balances {:?}", err))?;
+        self.previous_epoch_participation
+            .push(0b00000000)
+            .map_err(|err| anyhow!("Couldn't push to previous_epoch_participation {:?}", err))?;
+        self.current_epoch_participation
+            .push(0b00000000)
+            .map_err(|err| anyhow!("Couldn't push to current_epoch_participation {:?}", err))?;
```

Comment:
```suggestion
        self.previous_epoch_participation
            .push(0)
            .map_err(|err| anyhow!("Couldn't push to previous_epoch_participation {:?}", err))?;
        self.current_epoch_participation
            .push(0)
            .map_err(|err| anyhow!("Couldn't push to current_epoch_participation {:?}", err))?;
```

the extra 0's aren't need

Reply:
Done at https://github.com/ReamLabs/ream/pull/103/commits/b1628f6ef5643231367ad65522a34d4bab06f3f6